workers/unity/Packages/com.improbable.gdk.buildsystem/WorkerBuilder.cs

```diff
@@ -49,7 +49,7 @@ public static void Build()
 
                 var workerTypesArg =
                     CommandLineUtility.GetCommandLineValue(commandLine, BuildWorkerTypes,
-                        "UnityClient,UnityGameLogic");
+                        "UnityClient,UnityGameLogic,AndroidClient,IosClient");
```

Comment:
Do we want to build all of these by default? This will double build times for users where this may only be applicable to a small portion of them

Reply:
no, we don't. default should not include mobile

Reply:
(y)